haskell-language-server.cabal

```diff
@@ -83,18 +83,11 @@ library
   default-extensions: DataKinds, TypeOperators
 
 -- Plugin flags are designed for 'cabal install haskell-language-server':
--- - Packaged plugins should be manual:False
--- - Non packaged plugins and bulk flags should be manual:True
 -- - Bulk flags should be default:False
 -- - Individual flags should be default:True
 
-flag all-plugins
-  description: Enable all non formatter plugins
-  default:     False
-  manual:      True
-
-flag all-formatters
-  description: Enable all fomatters
+flag force-plugins
+  description: Force the inclusion of plugins even if they are not buildable by default with a specific ghc version
```

Comment:
Hmm, from the name/description I would also expect this to enable all the plugins. In fact it just removes a constraint. So maybe we could call it something like `ignore-ghc-bounds`?

Reply:
yeah you are right: we have the rename plugin disabled in the flag and i would expect the flag enable it

what about `ignore-plugins-ghc-bounds`?

Reply:
Sounds good, verbose names are fine IMO